packages/contracts/src/utils/web3_wrapper.ts

```diff
@@ -45,9 +47,29 @@ const providerConfigs = testProvider === ProviderType.Ganache ? ganacheConfigs :
 
 export const provider = web3Factory.getRpcProvider(providerConfigs);
 const isCoverageEnabled = env.parseBoolean(EnvVars.SolidityCoverage);
+const isProfilerEnabled = env.parseBoolean(EnvVars.SolidityProfiler);
+if (isCoverageEnabled && isProfilerEnabled) {
+    throw new Error(
+        `Unfortunately for now you can't enable both coverage and profiler at the same time. They both use coverage.json file and there is no way to configure that.`,
```

Comment:
Is this because they both use istanbul under-the-hood?

Reply:
Yes. That's because they both write to `coverage/coverage.json` file and Istanbul reads only from it